src/components/lists/List/index.js

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
-import styles from './lists.module.scss';
+import styles from '../lists.module.scss';
 
 function List ( { type, children } ) {
   return (
-    <div className={ styles[type] }>{children}</div>
+    <ul className={ styles[type] }>
+      {children}
```

Comment:
This would require a dev to then add `<li>` as children. I'm thinking that we should make this easier for them and have this maybe be the default unless they pass in an array of text as a prop, in which case the component would build itself with that given array.